seqr/views/apis/variant_search_api_tests.py

```diff
@@ -275,6 +285,8 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts, mock_erro
         response_json = response.json()
         self.assertSetEqual(set(response_json.keys()), set(EXPECTED_SEARCH_RESPONSE.keys()))
         self.assertDictEqual(response_json, EXPECTED_SEARCH_RESPONSE)
+        lirical_data = response_json['phenotypeGeneScores']['I000001_na19675']['ENSG00000268903']['lirical']
+        self.assertListEqual(sorted(lirical_data, key=lambda d: d['diseaseId']), EXPECTED_LIRICAL_DATA)
```

Comment:
just have the lirical data be sorted in the response so you can deterministically test this

Reply:
I'm hesitant to add sorting to the API just for testing.

Reply:
We do it all over the place in the code. It is better to have deterministic responses anyways even if the client code doesn't really on it, because it's better for the same query to always return the same response. You can use an order statement on the database query itself to keep the order deterministic here